### PR TITLE
Mining Speed Buff PR

### PR DESCRIPTION
**Pull request by Rexdyer1**
## About The Pull Request

This PR rebalances lots of aspects of mining, it changes the general drill speed of asteroid turf, updates the statistics for the pickaxes upgrading them to a faster speed, and updates stats for mech drills to make them more powerful: Drill's heads are more durable and drilling sand only gives 1/10th the degradation of the drill. Plasma cutter mech attachment now pierces multiple tiles per blast.  

## Why It's Good For The Game

These changes are to speed up the gameplay for mining and increasing value for mining drills beyond just being good for selling on the beacon. One star tools are put above their counterparts and on par with tools 1 tier above them combined with their original speed, which makes using them much more worth it. The mech is underwhelming to say the least. its way to clunky and its tools are relatively bad at collecting resources; with these updates to the drills and plasma cutter it should make mining mechs a serious force to be reckoned with and not just something that has no use. I considered just bringing up the stats for all drills up to like 80 each or something. but that isn't the most effective, so I just changed turf dig speed to fast. Basically this makes mining far more fast paced. And while this may compete slightly with Golem mining, I think the easy variety of the giant drill offers competition against the ability to drill fast, it also makes using a hand drill in general more tolerable. 

tl;dr: speed up hand drills to make them tolerable. hopefully if this works people will actually buy materials from the guild instead of just having it all bought from the trade beacon by the captain or given by the eye of the protector. if we have miners they should actually have some use. btw I'm gonna try and port the auto-pick up options from other servers like mechs and pouches on TG have....that'll be fun to figure out. 
## Testing

I drilled the rocks with the drills. everything appeared in order. 

## Changelog
:cl:
Balance: changed drill stats all across the board
tweak: overall turf dig speed
balance: Mech drill effectiveness
Balance:Mech plasma cutter effectiveness
/:cl:

### CODE REVIEW

**Comment by Rexdyer1** [comment]
Well let me know what ya think. hopefully with this PR we can finally see miners able to grab resources in a reasonable time frame and be able to sell them to the crew.

**Comment by Rexdyer1** [comment]
I decided to buff the proccessing a little bit to keep the waiting down

**Review by TheShown911 — APPROVED** [review]
